scripts/storybook/index.d.ts

```diff
@@ -0,0 +1 @@
+export { getPackageStoriesGlob, getCodesandboxBabelOptions, loadWorkspaceAddon } from './utils';
```

Comment:
this provides valid type definitions when the module is being consumed from `.ts` file without need of manual duplication of type definitions that gets out of sync rather quickly